### PR TITLE
Update map gl

### PR DESCRIPTION
**Pull request by elric-wamugu**
fixes #

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Comment by gabriel-piles** [comment]
I don't know if it is part of this task, but the map is not updating when changing coordinates values:

output

**Comment by elric-wamugu** [comment]
Hello @gabriel-piles, that seems to be a different maps issue not connected to this map module update as tested on a branch with an older version of mapbox

File: yarn.lock [hunk 1]
```diff
@@ -2351,6 +2317,8 @@ braces@^2.3.0, braces@^2.3.1, braces@^2.3.2:
     split-string "^3.0.2"
     to-regex "^3.0.1"
 
+<<<<<<< HEAD
+=======
```
Comment (RafaPolit) [inline yarn.lock]: There is a commit of a git conflict on the package json!

The actual 

```
<<<<<<< HEAD
=======
```

And 

```
>>>>>>> development
```

got COMMITTED into file.  Please, resolve conflicts... do not commit the automatically generated conflict lines.

File: yarn.lock [hunk 2]
```diff
@@ -2368,6 +2336,7 @@ brfs@^1.4.0:
     static-module "^2.2.0"
     through2 "^2.0.0"
 
+>>>>>>> development
```
Comment (RafaPolit) [inline yarn.lock]: Here as well

File: yarn.lock (hunk 1 above)
Comment (elric-wamugu) [reply yarn.lock]: Hey @RafaPolit, this is resolved now

**Comment by elric-wamugu** [comment]
@RafaPolit The reason why I was not catching the map.onHover js error is because I use Mozilla Firefox as my primary browser and that error was not being captured in the dev tools. The error is now fixed and tested on Chrome

**Comment by RafaPolit** [comment]
@elric-wamugu please ensure that all checks are green before assigning ready for QA.  We cannot merge a PR with failing unit_tests.

**Comment by elric-wamugu** [comment]
@RafaPolit The unit test that failed was the intermittent Out of Memory issue by our Circle CI pipeline after updating the branch by a merge from the development branch. Rerunning the tests again after a fresh update cleared the failing test. It's ready now.

**Comment by konzz** [comment]
@RafaPolit The map will throw an error when you edit an entity, remove the lat and lon values, and then save.

**Comment by elric-wamugu** [comment]
> @RafaPolit The map will throw an error when you edit an entity, remove the lat and lon values, and then save.
> Check if it happens with this map version.

@konz @RafaPolit I've tested with both map versions from the development branch and this new maps module branch and the issue is happening in both. When you set the coordinates by dragging the marker it raises the error, after clicking on the clear coordinates link and setting the coordinate values again via the map marker it saves/updates successfully. See attached screen grab 
Peek 2020-02-05 20-27

**Pull request merged**